app/policies/partner_policy.rb

```diff
@@ -37,6 +37,10 @@ def destroy?
     return true if user.only_neighbourhood_admin_for_partner?(record.id)
   end
 
+  def clear_address?
+    index?
```

Comment:
i know that logically this will return the same result right now, but I think you should use `edit?` here instead of `index?` as the defining logic for who gets to edit this, as this should really be the deciding scope, and it will get changed in future!

Reply:
I did think after submitting this should be `update?`